app/views/hub/clients/show.html.erb

```diff
@@ -124,68 +124,68 @@
+              <div class="client-profile__field-group client-recovery-rebate-credit-amount">
+                <% if @client.intake.eip1_amount_received.present? && @client.intake.eip2_amount_received.present? %>
+                  <div class="field-display">
+                    <span class="form-question"><%= t("hub.recovery_rebate_credit_amount.amount_1") %>:</span>
+                    <span class="label-value">$<%= @client.intake.eip1_amount_received %> </span>
+                  </div>
+                  <div class="field-display">
+                    <span class="form-question"><%= t("hub.recovery_rebate_credit_amount.amount_2") %>:</span>
+                    <span class="label-value">$<%= @client.intake.eip2_amount_received %> </span>
+                  </div>
+                <% end %>
+                <% if !@client.archived? %>
                   <div class="field-display">
-                    <span class="form-question"> Outstanding RRC / Claimed RRC: </span>
-                    <span class="label-value"><%= "$#{benefits.outstanding_recovery_rebate_credit}" if benefits.outstanding_recovery_rebate_credit %></span>
-                    /
-                    <span class="label-value"><%= "$#{benefits.claimed_recovery_rebate_credit}" if benefits.claimed_recovery_rebate_credit %></span>
+                    <span class="form-question"><%= t("hub.recovery_rebate_credit_amount.amount_3") %>:</span>
+                    <span class="label-value">$<%= @client.intake.eip3_amount_received || 0 %> </span>
                   </div>
-                  <% unless @client.intake.advance_ctc_amount_received.nil? %>
+                <% end %>
+                <% if @client.intake.drop_off? && @client.intake.eip1_and_2_amount_received_confidence.present? %>
+                  <div class="field-display">
+                    <span class="form-question">Confidence:</span>
+                    <span class="label-value"><%= @client.intake.eip1_and_2_amount_received_confidence.try(:capitalize) %> </span>
+                  </div>
+                <% end %>
+                <% unless @client.intake.drop_off? %>
+                  <% tax_return = @client.tax_returns.find_by(year: TaxReturn.current_tax_year) %>
+                  <% if tax_return %>
+                    <% benefits = Efile::BenefitsEligibility.new(tax_return: tax_return, dependents: @client.intake.dependents) %>
                     <div class="field-display">
-                      <span class="form-question">Outstanding CTC / Advance CTC Received:</span>
-                      <span class="label-value"><%= "$#{benefits.outstanding_ctc_amount}" if benefits.outstanding_ctc_amount %></span>
+                      <span class="form-question"> Outstanding RRC: </span>
+                      <span class="label-value">$<%= benefits.outstanding_recovery_rebate_credit || 0 %></span>
+                      <% case @client.intake.claim_owed_stimulus_money.capitalize %>
```

Comment:
I wonder if we can do something like this 
```
<% status_background_color_class = intake.claim_owed_stimulus_money_yes? ? "background--green" : (intake.claim_owed_stimulus_money_no? ? "background--red" : "background--grey") %>
                      <span class="label-value client-profile__status-label client-profile__status-label <%= status_background_color_class %>"><%= @client.intake.claim_owed_stimulus_money.capitalize %></span>
```
instead of this case statement